api.js

```diff
@@ -955,7 +970,7 @@ const checkMeetingIntersect = function(meetingObj, timeSlotObj) {
     }
 };
 
-const getSuggestions = function(userTimes) {
+const getSuggestions = function(userTimes, duration) {
```

Comment:
duration defaults to 1hr, but it looks like we are passing to getSuggestions( ) but never reading it.  Does this logic need the meeting duration value?

Reply:
I was planning on doing the duration logic there, but didn't complete it yet